homeassistant/components/jellyfin/config_flow.py

```diff
@@ -51,9 +51,6 @@ async def async_step_user(
         self, user_input: dict[str, Any] | None = None
     ) -> ConfigFlowResult:
         """Handle a user defined configuration."""
-        if self._async_current_entries():
-            return self.async_abort(reason="single_instance_allowed")
```

Comment:
You should be able to remove the translation key as well since it uses the homeassistant domain

Reply:
done thanks